### PR TITLE
API Documentation Improvements

### PR DESCRIPTION
**Pull request by ofelcan164**
All pages have been updated to be more cohesive and simple to understand.

### CODE REVIEW

File: sections/file_types.md [hunk 1]
```diff
@@ -1,66 +1,67 @@
...
+Each file in Image Relay is assigned a file type. Different file types have different sets of metadata terms associated with them.
 
-Note that on the web, this is known as a Metadata Template. Not all plans have the ability to create new templates and alter the terms associated with your default template. Please contact support@imagerelay.com if you have questions about your plan features.
+_**Note:** On the web, file types are referred to interchangeably with Metadata Templates. Not all plans have the ability to create new templates and alter the terms associated with your default template. Accounts require `Admin Access to Metadata Tools` for this functionality to be accessible. Please contact support@imagerelay.com if you have questions about your plan features._
 
 Get File Types
 --------------
 
-* `GET /file_types.json` will return all the file types for the account.
+* `GET /file_types.json` will return all the file types for the authenticated user.
```
Comment (mdoyle13) [inline sections/file_types.md:11]: I don't think this is actually true. I believe its for the client

Comment (mdoyle13) [reply sections/file_types.md:11]: line 8 of asset types controller is `current_user.client.metagroups.with_sorted_metaterms_and_options.order(:name))`

File: sections/keywording.md [hunk 2]
```diff
@@ -3,68 +3,69 @@ Keywording
...
 
-Keyword sets are groups of keywords that can be associated with an asset.
+Keyword/Tag Sets
+============
+Keyword sets are groups of keywords that can be associated with an asset
 
 Get Keyword Sets
 ----------------
-* `GET /keyword_sets.json` will return `200 OK` and the API user's list of keyword sets.
+* `GET /keyword_sets.json` will return `200 OK` and the authenticated user's list of keyword sets.
```
Comment (mdoyle13) [inline sections/keywording.md:15]: I believe this returns the client's keyword sets if the user has permission for metadata

File: sections/uploads.md [hunk 3]
```diff
@@ -1,82 +1,75 @@
 
-Uploads
+File Uploads
 =======
 
-The upload API is how you can add files to Image Relay. Files are uploaded to Image Relay through the creation of upload jobs. And files are sent up in chunks.
+The upload API is how you can add files to Image Relay. Files are uploaded to Image Relay through the creation of upload jobs, and then individual files are sent up in chunks.
```
Comment (justinperkins) [inline sections/uploads.md:5]: @ofelcan164 can you add a sentence here that describes nested files along with a link to the new Nested Files page?

**Review by justinperkins — CHANGES_REQUESTED** [review]

File: sections/file_types.md (hunk 1 above)
Comment (ofelcan164) [reply sections/file_types.md]: @mdoyle13 I pushed the change. Let me know if that is what you were thinking.

File: sections/keywording.md (hunk 2 above)
Comment (ofelcan164) [reply sections/keywording.md]: @mdoyle13 Done.

File: sections/uploads.md (hunk 3 above)
Comment (ofelcan164) [reply sections/uploads.md:5]: @justinperkins Added a a bit there. Let me know if that looks good.

File: sections/file_types.md (hunk 1 above)
Comment (mdoyle13) [reply sections/file_types.md]: @ofelcan164 I think you forgot to push the changes. I'm not seeing them.

**Review by mdoyle13 — CHANGES_REQUESTED** [review]

File: sections/file_types.md (hunk 1 above)
Comment (ofelcan164) [reply sections/file_types.md]: Fairly confident I did, and when I push again it says everything is up to do. So I am confused.
This is the link to the commit I made yesterday @mdoyle13 https://github.com/imagerelay/API/pull/34/commits/12d639d2497f877c025c209c33a82f1de7d75178.

Comment (mdoyle13) [reply sections/file_types.md]: That's odd. I see the changes now. Github may be having caching issues.

**Review by mdoyle13 — APPROVED** [review]
I'm good with this. Let's wait for @justinperkins approval before merging into master

**Review by justinperkins — APPROVED** [review]

**Pull request merged**